### PR TITLE
Emily2

### PR DESCRIPTION
**Pull request by Emily484**
post comment

### CODE REVIEW

File: client/src/actions/postActions.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const getSinglePost = (postId, history) => dispatch => {
 // Post Comment
 export const sendComment = comment => dispatch => {
   axios
-    .post("/api/post/comment/${postid}", comment)
+    .post(`/api/post/comment/${postid}`, comment)
```
Comment (slpriya) [inline client/src/actions/postActions.js]: postId is  missing. It is not passed as parameter.

**Review by slpriya — COMMENTED** [review]
pls check comment

File: client/src/actions/postActions.js (hunk 1 above)
Comment (Emily484) [reply client/src/actions/postActions.js]: I updated it, I hadn't pushed the changes I made in postActions.js

**Review by slpriya — APPROVED** [review]
Looks Good

**Pull request closed without merging**